pkgs/by-name/se/sequoia-sq/package.nix

```
@@ -74,10 +70,10 @@ rustPlatform.buildRustPackage rec {
   passthru.updateScript = nix-update-script { };
 
   meta = {
-    description = "Cool new OpenPGP implementation";
+    description = "A command line application exposing a useful set of OpenPGP functionality for common tasks.";
```

Perhaps it should be.

```suggestion
    description = "Command line application exposing a useful set of OpenPGP functionality for common tasks";
```

Because of https://github.com/NixOS/nixpkgs/blob/9829e28e9b7b5788e34f157d9024840f38888e87/pkgs/README.md#L408-L411